copt/utils.py

```diff
@@ -333,14 +333,18 @@ class SquareLoss:
   The Squared loss is defined as
 
   .. math::
-      \frac{1}{n}\|A x - b\|^2~,
+      \frac{1}{2n}\|A x - b\|^2~,
 
   where :math:`\|\cdot\|` is the euclidean norm.
   """
 
-    def __init__(self, A, b, alpha=0):
+    def __init__(self, A=None, b=None, alpha=0):
+        if A is None and b is None:
+            raise ValueError("Please specify A or b.")
         if A is None:
             A = sparse.eye(b.size, b.size, format="csr")
+        if b is None:
+            b = 0
```

Comment:
please revert and we'll discuss how to deal with missing inputs in a different PR